### PR TITLE
add popup-modal

### PR DESCRIPTION
**Pull request by mazmassa**
We are adding a `<PopUpModal>` modal component.

Screenshot 2023-05-17 at 11 39 47

### CODE REVIEW

**Review by maxime-carabina — APPROVED** [review]

File: src/components/PopupModal/PopupModal.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    _onClose?.();
+  }
+
+  if (children) {
+    return (
+      <div
+        data-testid="popup-modal-header"
+        className="flex items-start justify-between p-4 rounded-t"
+      >
+        <h3 className="text-xl font-semibold text-f-primary">{children}</h3>
```
Comment (pivilartisant) [inline src/components/PopupModal/PopupModal.tsx:41]: ` font-semibold ` do we not have a corresponding font in the presets ?

**Review by pivilartisant — APPROVED** [review]
left a comment for clarity but lgtm :)

File: src/components/PopupModal/PopupModal.tsx (hunk 1 above)
Comment (pivilartisant) [inline src/components/PopupModal/PopupModal.tsx:41]: ` font-semblod ` do we not have corresponding presets in the design system ?

Comment (mazmassa) [reply src/components/PopupModal/PopupModal.tsx:41]: Good catch! 

In fact we don't need that since is a children and we can customized that!

Comment (maxime-carabina) [reply src/components/PopupModal/PopupModal.tsx]:
```typescript
".mas-h3": {
          fontSize: "14px",
          fontWeight: "500",
          fontFamily: theme("fontFamily.Urbane"),
          lineHeight: "16px",
        },
``` 

Sure! We have a className for `h3` tag.

Comment (maxime-carabina) [reply src/components/PopupModal/PopupModal.tsx]:
```typescript
".mas-h3": {
          fontSize: "14px",
          fontWeight: "500",
          fontFamily: theme("fontFamily.Urbane"),
          lineHeight: "16px",
        },
``` 

Sure! We have this for `h3` tag.

Comment (mazmassa) [reply src/components/PopupModal/PopupModal.tsx]: Just updated! We are rendering {children}. So no need that conf. It's up to the user to add that.

Comment (maxime-carabina) [reply src/components/PopupModal/PopupModal.tsx]: Ok!

**Review by pivilartisant — APPROVED** [review]

File: src/components/PopupModal/PopupModal.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    console.log("event on open");
+  },
+  onClose: () => {
+    console.log("event on close");
+  },
+};
+
+export const _Regular = {
+  render: () => (
+    <div className="h-[450px]">
```
Comment (pivilartisant) [inline src/components/PopupModal/PopupModal.stories.tsx:21]: is this value used to be pixel perfect with design ?  Shouldn't we use `w-full` when we do our integrations ?

**Pull request merged**